docs/STUMP_Tiles_Algorithm_for_Cache_Optimization.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Instead of <code>100_000</code>  groups and <code>100_000</code>  elements, I would purposely change it to something visually more distinct like <code>50_000</code>  groups and <code>200_000</code>  elements. This way, when the reader sees these numbers in your code, it is easy to distinguish if you are referring to "groups" or "elements"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>